static/src/js/reducers/__tests__/timeline.test.js

```diff
@@ -94,3 +95,13 @@ describe('RESTORE_FROM_URL', () => {
     expect(timelineReducer(undefined, action)).toEqual(expectedState)
   })
 })
+
+describe('TOGGLE_TIMELINE', () => {
```

Comment:
This action type is no longer used in the timeline reducer, this test should be removed (and the import on line 6)